scripts/mkcloud

```diff
@@ -230,7 +232,8 @@ function sshrun()
 EOF
     env|grep -e ^debug_ -e ^pre_ -e ^want_ -e ^net_ -e ^nodenumber -e ^clusterconfig | sort >> $mkcconf
 
-    $scp $qa_crowbarsetup $mkcconf root@$adminip:
+    #TODO TH: discuss if the file extension should be part of the variable or not (like it is now)
```

Comment:
imho it shouldn't be, and the files should probably be in some subdirectory that is cloud version specific..